polyfills/Object/values/config.json

```diff
@@ -0,0 +1,27 @@
+{
+	"aliases": [
+		"es2017"
```

Comment:
Actually I just copy the `config.json` from [Object.entries](https://github.com/Financial-Times/polyfill-service/blob/master/polyfills/Object/entries/config.json).